tools/dbslower.py

```diff
@@ -101,17 +174,61 @@
 }
 """
 
-usdts = map(lambda pid: USDT(pid=pid), args.pids)
-for usdt in usdts:
-    usdt.enable_probe("query__start", "probe_start")
-    usdt.enable_probe("query__done", "probe_end")
+if args.path and not args.pids:
+    # Uprobes mode
+    program = program_uprobe
+
+    symbols = subprocess.check_output(["nm", "-aD", args.path])
```

Comment:
We have built-in symbol support in the BPF module. Can you see if that can be used here, instead of invoking nm directly?